aicsimageio/readers/czi_reader.py

```diff
@@ -95,7 +95,7 @@ def _is_supported_image(fs: AbstractFileSystem, path: str, **kwargs: Any) -> boo
                 )
 
             with fs.open(path) as open_resource:
-                CziFile(open_resource)
+                CziFile(open_resource.f)
```

Comment:
I would suggest making this a really small function that returns `resource.f` so that the actual `.f` part only shows up in one place.  It's a minor thing that could help hide that implementation detail.
for example:
```
def get_file_handle(fs_local_opener):
    return fs_local_opener.f
```

Reply:
or maybe get_buffered_reader or whatever

Reply:
I am going to merge without this. Returning an open buffer means we may have a open file handle down the line.

Would need to do something like:

```python
def get_file_handle(fs, path):
    yield fs.open(path).f
```

but that also doesn't ensure a closed handle after we are done.

Reply:
I'm fine w/that, but what's the difference between 
```
with fs.open(path) as file:
    CziFile(function_that_returns_file_dot_f(file))
```
and 
```
with fs.open(path) as file:
    CziFile(file.f)
```
?

Reply:
I wasn't saying to wrap the `with` part in a function. only the bit that calls `.f`

Reply:
Ahhhh that was my confusion. I thought you were asking for the `with` / context manager to be included. My mistake.